ios/RCTWKWebView/RCTWKWebView.m

```diff
@@ -354,6 +372,26 @@ - (void)webView:(__unused WKWebView *)webView didFailProvisionalNavigation:(__un
 
 - (void)webView:(WKWebView *)webView didFinishNavigation:(__unused WKNavigation *)navigation
 {
+  if (_messagingEnabled) {
```

Comment:
Have you considered using a user script here instead?

Reply:
This is just taken from here: https://github.com/facebook/react-native/blob/2546c95ac8eeca5a0056d34188a10f75a4b21a44/React/Views/RCTWebView.m#L301